config/software/stunnel.rb

```diff
@@ -43,13 +31,42 @@
   source sha256: "09ada29ba1683ab1fd1f31d7bed8305127a0876537e836a40cb83851da034fd5"
 end
 
+version "5.39" do
+  source sha256: "288c087a50465390d05508068ac76c8418a21fae7275febcc63f041ec5b04dee"
+end
+
 build do
   env = with_standard_compiler_flags(with_embedded_path)
-  configure_string = "./configure --with-ssl=#{install_dir}/embedded --prefix=#{install_dir}/embedded"
-  if fips_enabled
-    configure_string += " --enable-fips"
+
+  patch source: "stunnel-on-windows.patch", plevel: 1
+
+  configure_args = [
+    "--with-ssl=#{install_dir}/embedded",
+    "--prefix=#{install_dir}/embedded",
+  ]
+  configure_args << "--enable-fips" if fips_mode?
+
+  configure(*configure_args, env: env)
+
+  if windows?
+    # We only build the 64 bit executable for now
+    # make "mingw", env: env, cwd: "#{project_dir}/src"
+
+    # src/mingw.mk hardcodes and assumes SSL is at /opt so we patch and use
+    # an env variable to redirect it to the correct location
+    env["WIN32_SSL_DIR_PATCHED"] = "#{install_dir}/embedded"
+    make "mingw64", env: env, cwd: "#{project_dir}/src"
+
+    # Stack Smash Protection
+    arch_suffix = windows_arch_i386? ? "32" : "64"
+    windows_path = "C:/opscode/omnibus-toolchain/embedded/bin/mingw#{arch_suffix}/bin/libssp-0.dll"
```

Comment:
I couldn't find an example of a better way to setup copying these DLLs since we switched from `C:/msys`...